build.gradle

```diff
@@ -165,11 +174,16 @@ if (project.hasProperty('deployment')) {
       }
 
       signing {
+        useGpgCmd()
+        allprojects { ext."signing.gnupg.executable" =  System.getenv('GPG_EXECUTABLE') }
+        allprojects { ext."signing.gnupg.keyName" = System.getenv('GPG_KEYNAME') }
+        allprojects { ext."signing.gnupg.passphrase" = System.getenv('GPG_PASSPHRASE') }
```

Comment:
Using this the same as:
```
allprojects {
  ext."signing.gnupg.executable" =  System.getenv('GPG_EXECUTABLE')
  ext."signing.gnupg.keyName" = System.getenv('GPG_KEYNAME')
  ext."signing.gnupg.passphrase" = System.getenv('GPG_PASSPHRASE')
}
```

But with just one iteration?

Reply:
Yes!  Thank you.  e570439